src/Analyzers/CSharp/Analyzers/CSharpAnalyzers.projitems

```diff
@@ -144,4 +145,7 @@
   <ItemGroup Condition="'$(DefaultLanguageSourceExtension)' != '' AND '$(BuildingInsideVisualStudio)' != 'true'">
     <ExpectedCompile Include="$(MSBuildThisFileDirectory)**\*$(DefaultLanguageSourceExtension)" />
   </ItemGroup>
+  <ItemGroup>
+    <Folder Include="$(MSBuildThisFileDirectory)MakeStructMemberReadOnly\" />
+  </ItemGroup>
```

Comment:
📝 This shouldn't have been added.

Reply:
@tmeschter Potentially related to the other issues we're seeing with project system?